hive-bigquery-connector-common/src/main/java/com/google/cloud/hive/bigquery/connector/BigQueryMetaHookBase.java

```diff
@@ -389,22 +391,19 @@ public void preInsertTable(Table table, boolean overwrite) throws MetaException
   @Override
   public void commitInsertTable(Table table, boolean overwrite) throws MetaException {
     String engine = HiveConf.getVar(conf, HiveConf.ConfVars.HIVE_EXECUTION_ENGINE).toLowerCase();
-    if (engine.equals("mr")) {
-      // In Hive v3, `commitInsertTable()` never gets called for MR -- only for Tez.
-      // But in Hive v2, it does get called. So we exit here since the BigQueryOutputCommitter
-      // is already called automatically for MR.
-      return;
-    }
-    try {
-      JobDetails jobDetails = JobDetails.readJobDetailsFile(conf, HiveUtils.getDbTableName(table));
-      BigQueryOutputCommitter.commit(conf, jobDetails);
-    } catch (IOException e) {
-      throw new RuntimeException(e);
-    } finally {
-      // deleteOnExit in case of other jobs using the same workdir
-      JobUtils.cleanNotFail(
-          () -> JobUtils.deleteQueryWorkDirOnExit(conf),
-          CleanMessage.DELETE_QUERY_TEMPORARY_DIRECTORY);
+    if (engine.equals("tez")) {
```

Comment:
add comment why tez and mr treated differently here?

Reply:
@yigress There's already a comment about that in the method's javadoc. Is that ok?